core/src/components/textarea/textarea.ios.scss

```diff
@@ -12,6 +12,7 @@
   --padding-end: 0px;
   --padding-bottom: #{$textarea-padding-bottom};
   --padding-start: 0px;
+  --highlight-height: 0px;
```

Comment:
This doesn't need to be defined since the highlight is defined in `md` but I kept it here in case we ever move the highlight to the base file.